docs/static/azure-module.asciidoc

```diff
@@ -50,128 +52,97 @@ These {kib} dashboards are available and ready for you to use. You can use them
 
 * *SQL DB Queries*. Info about SQL Database queries and performance. 
 
-==== Azure_event_hubs plugin
-
-The Azure module uses the `azure_event_hubs` plugin. Basic understanding of the
-plugin and options is helpful when you set up the Azure module. See
-{logstash-ref}/plugins-inputs-azure_event_hubs.html[azure_event_hubs plugin
-documentation] for more information about configurations and options. 
-
 [[azure-module-prereqs]]
 ==== Prerequisites
 
 Azure Monitor enabled with Azure Event Hubs and the Elastic Stack are required
 for this module.
 
 [[azure-elk-prereqs]]
-===== Elastic prereqs
+===== Elastic prerequisites
 
-Logstash, Elasticsearch, and Kibana should be installed and running. The
-products are https://www.elastic.co/downloads[available to download] and easy to
-install. 
+The instructions below assume that you have {ls}, {es}, and {kib} running locally. 
+You can also run {ls}, {es}, and {kib} on separate hosts. 
 
 The Elastic Stack version 6.4 (or later) is required for this module. 
 
-NOTE: Logstash, Elasticsearch, and Kibana must run locally. You can also run 
-Elasticsearch, Kibana and Logstash on separate hosts to consume data from Azure.
+The Azure module uses the `azure_event_hubs` input plugin to consume logs and
+metrics from your Azure environment. It is installed by default with {ls} 6.4
+(or later). Basic understanding of the plugin and options is helpful when you
+set up the Azure module.  
+See the {logstash-ref}/plugins-inputs-azure_event_hubs.html[azure_event_hubs input
+plugin documentation] for more information. 
+
+Elastic products are https://www.elastic.co/downloads[available to download] and
+easy to install. 
 
 [[azure-prereqs]]
-===== Azure prereqs
+===== Azure prerequisites
 
-Azure Monitor should be configured to stream logs to one or more Event Hubs. 
+Azure Monitor should be configured to stream logs to one or more Event Hubs.
```

Comment:
Can you link this to the [[azure-resources]] section below ?

Reply:
The link is in the next line down (line 84).  Do you want it both places?

Reply:
doh! sorry , i missed that.  good as-is